mysite/w5/gaming.py

```diff
@@ -8,30 +8,46 @@
 from utils.data_formatting import safer_convert, safer_math_log, mark_advice_completed
 from utils.logging import get_logger
 from flask import g as session_data
-from consts.consts_w5 import snail_max_rank
+from consts.consts_w5 import snail_max_possible_rank
 
 logger = get_logger(__name__)
 
 def getSnailResetTarget(snail_rank):
     if snail_rank == 0:
-        return 0  # Can't actually happen, but...
+        return 0 # Can't actually happen, but...
     else:
         return max(1, snail_rank - snail_rank % 5)
 
-# Cache the info since iteration is involved (up to ~0.035 seconds)
+# Cache the info since iteration is involved in the computation
 @functools.cache
 def getSnailLevelUpInfo(final_ballad_bonus, target_confidence_levels):
 
     def success_chance(snail_level, encouragement):
-        return min(1,
-                   (1 - 0.1 * snail_level ** 0.72) * (1 + 100 * encouragement / (25 + encouragement) / 100) * final_ballad_bonus)
+        if 24 < snail_level:
+            return min(1,
+                       (1 - 0.6 * (snail_level - 24) ** 0.16) * (1 + 50 * encouragement / (3 + encouragement) / 100) * final_ballad_bonus)
+        else:
+            return min(1,
+                       (1 - 0.1 * snail_level ** 0.71) * (1 + 110 * encouragement / (25 + encouragement) / 100) * final_ballad_bonus)
```

Comment:
```suggestion
        if snail_level < 25:
            return min(1,
                       (1 - 0.1 * snail_level ** 0.71) * (1 + 110 * encouragement / (25 + encouragement) / 100) * final_ballad_bonus)
        else:
            return min(1,
                       (1 - 0.6 * (snail_level - 24) ** 0.16) * (1 + 50 * encouragement / (3 + encouragement) / 100) * final_ballad_bonus)
```

i'd swap these branches so the reading order matches the logical order (level 1-24 first, then level 25+)

same applies to `reset_chance()` below